.github/workflows/ci.yml

```diff
@@ -306,6 +307,42 @@ jobs:
           name: airflow-backport-readmes
           path: "./files/airflow-backport-readme*"
 
+  prepare-provider-packages:
+    timeout-minutes: 30
+    name: "Provider packages"
+    runs-on: ubuntu-latest
+    needs: [build-info, ci-images]
+    env:
+      INSTALL_AIRFLOW_VERSION: "2.0.0-dev"   # Note that this causes local installation
+      PYTHON_MAJOR_MINOR_VERSION: ${{needs.build-info.outputs.defaultPythonVersion}}
+    if: github.repository == 'apache/airflow' || github.event_name != 'schedule'
+    steps:
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v2
+      - name: "Setup python"
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_MAJOR_MINOR_VERSION }}
+      - name: "Free space"
+        run: ./scripts/ci/tools/ci_free_space_on_ci.sh
+      - name: "Prepare CI image ${{env.PYTHON_MAJOR_MINOR_VERSION}}:${{ env.GITHUB_REGISTRY_PULL_IMAGE_TAG }}"
+        run: ./scripts/ci/images/ci_prepare_ci_image_on_ci.sh
+      - name: "Prepare & test provider packages"
+        run: ./scripts/ci/provider_packages/ci_prepare_and_test_provider_packages.sh
+      - name: "Upload provider package artifacts"
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: airflow-backport-packages
+          path: "./files/airflow-packages-*"
```

Comment:
```suggestion
          name: airflow-backport-packages
          path: "./files/airflow-packages-*"
```
```suggestion
          name: airflow-providers-packages
          path: "./files/airflow-packages-*"
```

Reply:
This step is to build the backports, so I don't think so @kaxil (i.e. we still want to build backports for now)

Reply:
Preparing backport is L274 . From L310 it is preparing providers package

Reply:
Yep. Fixing.